packages/integration-sdk-runtime/testEventEmitters.js

```diff
@@ -0,0 +1,27 @@
+require('./test/mocks/mockWriteJsonToPath');
```

Comment:
Are these files going to end up being bundled in when the packages are built? Can you ensure that this is ignored? It may make sense to rename this file to match our same test naming convention, but ignore the file with jest (e.g. `testEventEmitters.test.ts`)

It would also be great if we could make all of these files TypeScript and execute with `ts-node` instead.

Reply:
I see there is a `/test` directory. Maybe we should move this into there instead of at the top-level of the package?

Reply:
Yeah, I had initially put this file in the root because I knew I would need to traverse from `test` to `src` (or `dist`) and I thought I would have had module resolution issues if my entrypoint was within `test`. But it seems that since I invoke `node` (or `ts-node`) from the root, the entrypoint is high enough to not cause resolution issues.

Resolved this in an incoming commit.